io.openems.edge.evcs.goe.chargerhome/src/io/openems/edge/evcs/goe/chargerhome/GoeChargerHomeImpl.java

```diff
@@ -0,0 +1,340 @@
+				}
+				
+				this.channel(GoeChannelId.SERIAL).setNextValue(json.get("sse").getAsString());
+				this.channel(GoeChannelId.FIRMWARE).setNextValue(json.get("fwv").getAsString());								
+				this.channel(GoeChannelId.CURR_USER).setNextValue(this.activeCurrent);
+				this.channel(GoeChannelId.STATUS_GOE).setNextValue(json.get("car").getAsInt());			
+				this.channel(Evcs.ChannelId.STATUS).setNextValue(ConvertGoeStatus(json.get("car").getAsInt()));				
+				this.channel(GoeChannelId.VOLTAGE_L1).setNextValue(nrg.get(0).getAsInt());
+				this.channel(GoeChannelId.VOLTAGE_L2).setNextValue(nrg.get(1).getAsInt());
+				this.channel(GoeChannelId.VOLTAGE_L3).setNextValue(nrg.get(2).getAsInt());
+				this.channel(GoeChannelId.CURRENT_L1).setNextValue(nrg.get(4).getAsInt()*100);
+				this.channel(GoeChannelId.CURRENT_L2).setNextValue(nrg.get(5).getAsInt()*100);
+				this.channel(GoeChannelId.CURRENT_L3).setNextValue(nrg.get(6).getAsInt()*100);
+				this.channel(GoeChannelId.ACTUAL_POWER).setNextValue(nrg.get(11).getAsInt()*10);
+		
+				this.channel(Evcs.ChannelId.PHASES).setNextValue(phases);				
+				this.channel(Evcs.ChannelId.MAXIMUM_POWER).setNextValue(GetMaxPower(phases));	
+				this.channel(Evcs.ChannelId.MINIMUM_HARDWARE_POWER).setNextValue(GetMinHardwarePower(phases, cabelCurrent));	
+				this.channel(Evcs.ChannelId.MAXIMUM_HARDWARE_POWER).setNextValue(GetMaxHardwarePower(cabelCurrent));	
+				this.channel(Evcs.ChannelId.CHARGE_POWER).setNextValue(nrg.get(11).getAsInt()*10);
+				this.channel(Evcs.ChannelId.CHARGING_TYPE).setNextValue(ChargingType.AC);
+				
+				this.channel(GoeChannelId.ENERGY_TOTAL).setNextValue(json.get("eto").getAsInt()*100);				
+				this.channel(Evcs.ChannelId.ENERGY_SESSION).setNextValue(json.get("dws").getAsInt()*10/3600);
+			
+				this.channel(GoeChannelId.ERROR).setNextValue(err);
+				this.channel(Evcs.ChannelId.CHARGINGSTATION_COMMUNICATION_FAILED).setNextValue(false);
+				
+				this.setEnergySession();
+				this.setPower();				
+				
+			}
+			else{
+				this.channel(Evcs.ChannelId.CHARGINGSTATION_COMMUNICATION_FAILED).setNextValue(true);
+			}
+			
+			break;
+		}
+	}
+	
+	private int ConvertGoeStatus(int status) {
+		switch(status) {
+		case 1: // ready for charging, car unplugged
+			return Status.NOT_READY_FOR_CHARGING.getValue();
+		case 2: // charging
+			return Status.CHARGING.getValue();
+		case 3: // waiting for car
+			return Status.READY_FOR_CHARGING.getValue(); 
+		case 4: // charging finished, car plugged
+			return Status.CHARGING_FINISHED.getValue();
+		default:
+			return Status.UNDEFINED.getValue();
+		}
+	}
+	
+	private int ConvertGoePhase(int phase) {
+		int Phasen = (byte)phase & 0b00111000;			
+		switch(Phasen) {
+		case 8: // 0b00001000: Phase 1 is active
+			return 1;
+		case 24: // 0b00011000: Phase 1+2 is active
+			return 2;
+		case 56: // 0b00111000: Phase1-3 are active
+			return 3;
+		default:
+			return 0;
+		}
+	}
+	
+	private int GetMaxHardwarePower(int cableCurrent) {
```

Comment:
Sorry - It is not perfectly declared what the minimum Hardware Power and the maximum hardware power is and for what it is used.
Maybe we must change that generally, but for now, the minimum and maximum hardware power includes the power that can be set to the charging station in that situation.

This Channel must be set according to the minimum current, the used phases and the default voltage.
It is mainly used for the UI and the evcs Controller:

### Charging with one phase:
![grafik](https://user-images.githubusercontent.com/47855186/123094540-47506500-d42d-11eb-8774-3ddf1c1774f3.png)

### Charging with three phases:
![grafik](https://user-images.githubusercontent.com/47855186/123096575-8a133c80-d42f-11eb-9b6c-298143b223e7.png)

Normally we would write 6A on the left side and 32A on the right side - that would be much more easier.
But as the complete system and every Controller is working with power values and it is easier to understand that we have e.g. 12 kW from the PV and are charging these 12 kW in the car and so on ... we decided this way.

		int minHwCurrent = this.parent.config.minHwCurrent() / 1000;
		this.parent.channel(Evcs.ChannelId.MINIMUM_HARDWARE_POWER)
		.setNextValue(230 /* Voltage*/ * minHwCurrent /* min current*/ * phases);

This can be set when we read the relevant data.

In your case, the customer would be able to set 22kW even if the car is using one phase. 
Target of 22kW -> about 95A cannot be set to the EVCS and is confusing the customer in the Monitoring

Reply:
The minimum Hardware Power is 6A*230V * [Count of connected Phases]

The maximum Power is e.g. 11kW or 22kW (depends on the Model of the Charger) but the thing is a 11kW Charger can also set up to 32A but than the charger automaticly reduces to the max. of 16A